appveyor.yml

```diff
@@ -1,48 +1,48 @@
 environment:
-  COVERALLS_REPO_TOKEN:
-    secure: 2NJ5Ct55cHJ9WEg3xbSqCuv0rdgzzb6pnzOIG5OkMbTndw3wOBrXntWFoQrXiMFi
-    # this is pytest's token in coveralls.io, encrypted
-    # using pytestbot account as detailed here:
-    # https://www.appveyor.com/docs/build-configuration#secure-variables
-
   matrix:
-  # coveralls is not in the default env list
-  - TOXENV: "coveralls"
-  # note: please use "tox --listenvs" to populate the build matrix below
   - TOXENV: "linting"
+    PYTEST_NO_COVERAGE: "1"
   - TOXENV: "py27"
   - TOXENV: "py34"
   - TOXENV: "py35"
   - TOXENV: "py36"
   - TOXENV: "py37"
   - TOXENV: "pypy"
-  - TOXENV: "py27-pexpect"
+    PYTEST_NO_COVERAGE: "1"
   - TOXENV: "py27-xdist"
   - TOXENV: "py27-trial"
   - TOXENV: "py27-numpy"
   - TOXENV: "py27-pluggymaster"
-  - TOXENV: "py36-pexpect"
   - TOXENV: "py36-xdist"
   - TOXENV: "py36-trial"
   - TOXENV: "py36-numpy"
   - TOXENV: "py36-pluggymaster"
   - TOXENV: "py27-nobyte"
   - TOXENV: "doctesting"
   - TOXENV: "py36-freeze"
+    PYTEST_NO_COVERAGE: "1"
   - TOXENV: "docs"
+    PYTEST_NO_COVERAGE: "1"
```

Comment:
@nicoddemus 
Was the removal of pexpect intentional here?

Reply:
Yes, those jobs don't run on Windows:

```
(.env36) λ tox -e py27-pexpect
GLOB sdist-make: c:\pytest\setup.py
_____________________________________________ summary _____________________________________________
SKIPPED:  py27-pexpect: platform mismatch
  congratulations :)
```

So it doesn't make sense to queue the job just to skip it right at the beginning. 👍